### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/395be1b77e37804ac250e5852e8139328292f51d

### CODE REVIEW

**Comment by flodolo** [comment]
@stephaniehobson @craigcook 
Is the goal to have this localized? Currently, it will only be localized by Smartling languages (paid agency, which covers French and German but not much more).

Otherwise, it needs to be added here https://github.com/mozilla/bedrock/blob/main/l10n/configs/pontoon.toml#L253-L301

File: en/mozorg/about-m24.ftl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+m24-about-we-build-products = We build products that put you in control — like { -brand-name-firefox }, { -brand-name-fakespot } and more.
+m24-about-fund = Fund
+m24-about-we-fund-and = We fund and invest in the people and efforts taking technology, the internet and AI in the right direction.
+
+## News
+
+m24-about-read-more = Read more
+m24-about-read-the-latest = Read the latest { -brand-name-mozilla } news & articles
+m24-about-14-ai-projects = 14 AI projects to watch: { -brand-name-mozilla }’s first Builders Accelerator cohort kicks off
+m24-about-at-our-rise25 = At our Rise25 Awards, the future of AI is trustworthy, inclusive and accountable
```
Comment (peiying2) [inline en/mozorg/about-m24.ftl:36]: We need to add `Rise25` to the brands.ftl file per our discussion yesterday.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Per our previous discussions, this page is open to the communities too, so let's add an entry in the pontoon.toml file, and add `Rise25` to the brands.ftl file.

**Comment by stephaniehobson** [comment]
@peiying2 Ah sorry 🤦‍♀️ I did it for the homepage but not the about page.

**Comment by stephaniehobson** [comment]
@flodolo yes, any community can translate it.

Is this the right thing to add?
```
[[paths]]
    reference = "en/mozorg/home-24.ftl"
    l10n = "{locale}/mozorg/home-24.ftl"
```

**Comment by peiying2** [comment]
> @flodolo yes, any community can translate it.
> 
> Is this the right thing to add?
> 
> ```
> [[paths]]
>     reference = "en/mozorg/home-24.ftl"
>     l10n = "{locale}/mozorg/home-24.ftl"
> ```

You can follow the format and add the new page in alphabetic order: https://github.com/mozilla-l10n/www-l10n/blob/f0072ba27f0c7f3cdd6231c3979a6fe7008a6157/configs/pontoon.toml#L253

**Comment by flodolo** [comment]
> @flodolo yes, any community can translate it.
> 
> Is this the right thing to add?
> 
> ```
> [[paths]]
>     reference = "en/mozorg/home-24.ftl"
>     l10n = "{locale}/mozorg/home-24.ftl"
> ```

The file here is `en/mozorg/about-m24.ftl`, not `home-24.ftl`. Other than that, yes

**Comment by stephaniehobson** [comment]
[PR with fixes](https://github.com/mozilla/bedrock/pull/15584).

File: en/navigation_refresh.ftl [hunk 2]
```diff
@@ -20,7 +20,9 @@ navigation-refresh-blog = Blog
 navigation-refresh-our-mission = Our Mission
 navigation-refresh-our-work = Our Work
 navigation-refresh-mozilla-ai = { -brand-name-mozilla } AI
+navigation-refresh-mozilla-ai-v2 = { -brand-name-mozilla-ai }
 navigation-refresh-mozilla-ventures = { -brand-name-mozilla } Ventures
+navigation-refresh-mozilla-ventures-v2 = { -brand-name-mozilla-venture }
```
Comment (peiying2) [inline en/navigation_refresh.ftl:25]: Should be plural `{ -brand-name-mozilla-ventures }`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Mozilla Ventures is plural. Also are you going to commit the other new file?

**Review by peiying2 — APPROVED** [review]

**Pull request merged**